isaric/parsers/idor-brazil.toml

```diff
@@ -447,32 +466,67 @@
   combinedType = "set"
   excludeWhen = "none"
   fields = [
-    { field = "cloro_cmyn", values = { 1 = "Chloroquine/Hydroxychloroquine" } },
-    { field = "hidroxic_cmyn", values = { 1 = "Chloroquine/Hydroxychloroquine" } },
     { field = "antiviral_cmtrt", values = { 1 = "Ribavirin" } },
     { field = "antiviral_cmtrt", values = { 2 = "Lopinavir/Ritonvir" } },
     { field = "antiviral_cmtrt", values = { 3 = "Interferon alpha" } },
     { field = "antiviral_cmtrt", values = { 4 = "Interferon beta" } },
     { field = "antiviral_cmtrt", values = { 5 = "Neuraminidase inhibitor" } },
-    { field = "antiviral_cmtrt", values = { 6 = "Other" } },                        # Other antiviral
+    { field = "il6", values = { 1 = "IL6 inhibitor" } },
```

Comment:
@abhidg I think il6 shouldn't be included as an antiviral here, interleukin inhibitors are immunosuppressants in ATC